index.html

```diff
@@ -14,13 +15,14 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
```

Comment:
- [ ] You did a great job👌 at implementing the required changes♻️, but I noticed that you forgot to implement this change which was stated by the previous reviewer😊, so kindly put the button of bars inside the nav because this will make your project friendly💑 with SEO to read more information about SEO [Click Here](https://www.quanzhanketang.com/website/web_search.html)  ⤵️